install.ps1

```diff
@@ -0,0 +1,56 @@
+If (-NOT ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole(`
```

Comment:
@siadevelopment have a look at this http://www.workingsysadmin.com/bypassing-powershell-execution-policy/